e2e/client/playwright/monitoring.spiking.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('spiking and unspiking an article', async ({page}) => {
```

Comment:
I noticed that u separated the spikes-article from the custom-workspace... my test was based on removing article from the custom-workspace... if that's not needed, then we should remove this test because removing article from the 'desk' is already covered in old tests.

Reply:
we already discussed this yesterday - but I'll post here for the record that in case we have the same test in protractor and playwright - we remove the protractor one.